slackbot-guysgals-reasoning.md

```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
+
+That's not the MAIN reason why the slackbot auto response exists though. There's a bigger purpose. The purpose is to try and get people to be more aware about the language they use and to try and be more inclusive towards other humans. Regardless of Class, Race, Gender, Sex or other divides that humans love putting up.
+
+We want to be a loving and inclusive community, and we know that the community is made up of the individuals. Thus if the individuals are made more aware of how better to be inclusive to the group, the community as a whole becomes more inclusive.
+
+
+# What we're not trying to do
+
+This is NOT about policing language or stopping freedom of speech.
+It's about getting people to be a little bit more aware of the language they use and the implications thereof.
+
+# I don't agree with this, what now?
+
+Please don't feel excluded or shamed, there's been a huge amount of discussion on several channels about diversity, this slackbot and general inclusivity.
+If you want to change something about how it works, be that a copy change or anything, we really do welcome suggestions to refine the bot and are very open about it's flaws, limitations and aggravations. It is slightly limited at the moment (it's not a custom bot, simply an auto-responder).
+
+Some relevant channels to look at include:
+* `#meta-guys-response` for talking about this specific slackbot
+* `#diversity` - Discussion of ways and means of ensuring that groups of people that are under-represented in tech (gender, ethnicity, sexual orientation, ability, and class) do not feel excluded and are given every possible opportunity to be part of the community.
+* `#others` - Stories, advice and sympathy on being an "other" in tech
+* `#codeofconduct` - Chatting about the ZATech CoC - https://github.com/zatech/code-of-conduct
+* `#asktheadmins` - Want to ask the admins a question but not sure who they are or what's going on, this is the place
+
+# Some frequently asked questions 
+
+* **Isn't it gender neutral in its plural form?** Maybe in certain situations, but which situations those are is ambiguous, [here's an unscientific survey](https://jvns.ca/blog/2013/12/27/guys-guys-guys/) that shows that not everyone agrees if you find a better survey, please post it for discussion in the groups above. What is for certain is that there are members of our community that do not think it is gender neutral. 
```

Comment:
break into new sentence: 
"...that shows that not everyone agrees. If you find a better..."